packages/zowe-explorer/package.json

```diff
@@ -1713,18 +1713,40 @@
           "description": "%zowe.jobs.confirmSubmission%",
           "scope": "window"
         },
+        "zowe.commands.useIntegratedTerminals": {
+          "type": "boolean",
+          "default": true,
+          "description": "%zowe.commands.useIntegratedTerminals%",
+          "scope": "window"
+        },
         "zowe.commands.alwaysEdit": {
           "type": "boolean",
           "default": true,
-          "description": "Allow editing of commands before submitting",
+          "description": "%zowe.commands.alwaysEdit%",
           "scope": "window"
         },
-        "zowe.commands.history": {
+        "zowe.commands.mvs.history": {
+          "default": {
+            "persistence": true,
+            "history": []
+          },
+          "description": "%zowe.commands.mvs.history%",
+          "scope": "application"
+        },
+        "zowe.commands.tso.history": {
```

Comment:
We currently keep history for Data Sets, USS and Jobs in local storage. Should these be moved into local storage to keep all the "history settings" in one place?

Reply:
One thing I did notice was that if I attempt to Ctrl+C without thinking about it, the terminal gets closed. I am not sure if we can do anything about that, but it is something that we might want to address if we can.